Common/Data/Market/VolumeRenkoBar.cs

```diff
@@ -0,0 +1,143 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Data.Market
+{
+    /// <summary>
+    /// Represents a bar sectioned not by time, but by some amount of movement in volume
+    /// </summary>
+    public class VolumeRenkoBar : BaseData, IBaseDataBar
```

Comment:
Seems pretty similar to the existing `RenkoBar`, do we need both? can we make them inherit each other? or maybe we need some base abstract share class?

Reply:
Sure. I think a base class works better, or it would be weird intuitively if `RenkoBar` is inherited from `VolumeRenkoBar`.